packages/eslint-plugin/test/no-bad-reference.test.ts

```diff
@@ -1,134 +1,127 @@
-import { ESLintUtils } from "@typescript-eslint/utils";
+import { runTestsWithFixtures } from "./fixtureTester";
+import * as rule from "../src/rules/no-bad-reference";
 
-import * as noBadReference from "../src/rules/no-bad-reference";
-
-const ruleTester = new ESLintUtils.RuleTester({
-  parser: "@typescript-eslint/parser",
-});
-
-ruleTester.run("@definitelytyped/no-bad-reference", noBadReference, {
+runTestsWithFixtures("@definitelytyped/no-bad-reference", rule, {
```

Comment:
oh noooooo
now I see why you wanted baselines

Reply:
Yes, this is annoying and tedious. I know the ts-eslint folk have wanted to make VFS possible, but then we'd have to somehow use such a VFS in the helpers which sounds infeasible.